lib/engine.js

```diff
@@ -66,3 +65,17 @@ requestAnimationFrame(function gameLoop() {
 
   requestAnimationFrame(gameLoop);
 });
+
+function selectTile(event) {
+  currentTile = board.getClickedTile(event.clientX, event.clientY);
+}
+
+function assignSimpleTower() {
+  if (currentTile) {
+    currentTile.addTower('simple');
```

Comment:
Actually I tried it and answered my own question. Looks like it throws an error in the console. I'm going to see if I can fix this so it doesn't throw an error, but just does nothing.